resource-watch/resource-watch
# Refactor `tools` service to use Axios

### Overview
This PR refactors the `pages` service to comply with the new approach for all API services.

### Testing instructions
Go to http://localhost:9000/admin/pages and check all the endpoints involved (CRUD)

[Pivotal task](https://www.pivotaltracker.com/story/show/170486008)
